frontend/src/features/admin-form/share/ShareFormModal.tsx

```diff
@@ -203,10 +213,10 @@ export const ShareFormModal = ({
           padding-top: 5px;
         "
       >
-        Powered by <a href="${window.location.origin}" style="color: #999">Form</a>
+        ${t('embed.poweredBy')} <a href="${window.location.origin}" style="color: #999">Form</a>
```

Comment:
nit: would be awesome if you could interpolate the link component here!
this would allow the sentence structure to better fit the grammar of the language translation.